README.md

```diff
@@ -600,6 +601,17 @@ var schema = Joi.array().items(Joi.string().valid('not allowed').forbidden(), Jo
 var schema = Joi.array().items(Joi.string().label('My string').required(), Joi.number().required()); // If this fails it can result in `[ValidationError: "value" does not contain [My string] and 1 other required value(s)]`
 ```
 
+#### `array.ordered(enabled)`
+
+A flag to control the match **in order** of `type` with the array values where:
+- `type` - a **joi** schema object to validate each array item against. `type` can be an array of values, or multiple values can be passed as individual arguments.
```

Comment:
You're still documenting things that have little to do with this method, the type is not in the method's signature.